index.bs

```diff
@@ -4132,23 +4133,25 @@ keyword is used, this includes <code class="idl">entries</code>,
 For read–write maplikes, it also includes <code class="idl">clear</code>,
 <code class="idl">delete</code>, and <code class="idl">set</code> methods.
 
-Maplike interfaces must not
-have any [=interface members=]
-named "<code>entries</code>", "<code>forEach</code>",
-"<code>get</code>", "<code>has</code>",
-"<code>keys</code>", "<code>size</code>", or
-"<code>values</code>",
-or have any [=inherited interfaces=]
-that have [=members=] with these names.
-Read–write maplike interfaces must not
-have any [=attributes=]
-or [=constants=] named
-"<code>clear</code>", "<code>delete</code>",
-or "<code>set</code>", or have any [=inherited interfaces=]
-that have [=attributes=] or [=constants=] with these names.
-
-Note: Operations named "<code>clear</code>", "<code>delete</code>",
-or "<code>set</code>" are allowed on read–write maplike
+Maplike interfaces, their [=inherited interfaces=], and any [=interfaces=] that
```

Comment:
Is it worth factoring out the "interface, its inherited interfaces, and all interfaces that inherit from it" concept so it can be reused, instead of copy/pasting it?